project1/application.py

```diff
@@ -0,0 +1,115 @@
+import os
+import sys
+import time
+
+from flask import Flask, render_template, request, url_for,flash,session,redirect
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import create_engine, desc
+from register import *
+from sqlalchemy.orm import scoped_session, sessionmaker
+from datetime import datetime
+
+
+app = Flask(__name__)
+
+# Check for environment variable
+if not os.getenv("DATABASE_URL"):
+    raise RuntimeError("DATABASE_URL is not set")
+
+
+
+# Configure session to use filesystem
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.config["SESSION_PERMANENT"] = False
+app.config["SESSION_TYPE"] = "filesystem"
+engine = create_engine(os.getenv("DATABASE_URL"))
+Session(app)
+db.init_app(app)
+
+with app.app_context():
+	db.create_all()
+
+
+@app.route("/")
+def index():
+	if 'Username' in session:
```

Comment:
'Username' does not match the username as mentioned while storing the variable.

Reply:
yes, resolved.